lib/skywalking/util.lua

```diff
@@ -28,13 +38,22 @@ local split = function(str, delimiter)
     return t
 end
 
+-- source: https://github.com/apache/apisix/blob/master/apisix/core/string.lua#L58
```

Comment:
Codes could change from time to time, please use perm link.

```suggestion
-- copied from: https://github.com/apache/apisix/blob/b9e36d093bbdafb6d70d332b90679ad13724c2a1/apisix/core/string.lua#L58-L67
```